src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptionsExtensions.cs

```diff
@@ -55,7 +55,7 @@ public static Channel CreateChannel(this OtlpExporterOptions options)
             channelCredentials = ChannelCredentials.Insecure;
         }
 
-        return new Channel(options.Endpoint.Authority, channelCredentials);
+        return new Channel(options.Endpoint.Host, options.Endpoint.Port, channelCredentials);
```

Comment:
I think this change is otherwise ok, however, we are discussing dropping our dependency on the old gRPC library. If we do, this code will go away. See #4860